orte/tools/orterun/orterun.c

```diff
@@ -1147,7 +1147,8 @@ static int parse_globals(int argc, char* argv[], opal_cmd_line_t *cmd_line)
     /* Do we want a user-level debugger? */
 
     if (orterun_globals.debugger) {
-        run_debugger(orte_basename, cmd_line, argc, argv, orterun_globals.num_procs);
+        orte_show_help("help-orterun.txt", "no_debugger_launch_support", false);
```

Comment:
Perhaps we should `#if 0` out the call to `run_debugger()` with a big comment explaining how we anticipate this to be temporary, and a pointer to the https URL for open-mpi/ompi#1225, and how the `#if 0/#else/#endif` should be removed once open-mpi/ompi#1225 is fixed...?

I say this because the call to `run_debugger()` is not incorrect, and when someone finally fixes this issue, they'll undoubtedly have to look up what the Right Thing is to put back instead of `orte_show_help()`.